tools/reconciler/Makefile

```diff
@@ -9,7 +9,8 @@ deploy-reconciler:
 		--set global.component_reconcilers.enabled=true --set global.component_reconcilers.authentication=false \
 		--set compass.enabled=false --set mothership-reconciler.db.encryptionKey=5bc19d3a2032fb8795cd86e08b473a351631505d2522991266b1fb85f89bad5f \
 		--set mothership-reconciler.db.serviceHost=reconciler-postgresql --set mothership-reconciler.db.reconcilerUsername=postgres \
-		--set mothership-reconciler.db.reconcilerPassword=test --set postgresql.postgresqlPassword=test --set postgresql.persistence.enabled=false \
+		--set mothership-reconciler.db.reconcilerPassword=test --set mothership-reconciler.options.verbose=true \
+		--set postgresql.postgresqlPassword=test --set postgresql.persistence.enabled=false \
 		--set global.mothership_reconciler.auditlog.useTestConfig=true --set global.mothership_reconciler.auditlog.persistence.enabled=false \
 		--set postgresql.initdbScriptsConfigMap=reconciler-postgresql-db-init \
 		--set postgresql.persistence.enabled=true \
```

Comment:
oh, just figure out we have `postgresql.persistence.enabled=false` and `postgresql.persistence.enabled=true` ...